internal/pkg/workspace/workspace.go

```diff
@@ -268,14 +268,15 @@ func (ws *Workspace) summaryPath() (string, error) {
 
 func (ws *Workspace) createCopilotDir() error {
 	// First check to see if a manifest directory already exists
-	existingWorkspace, _ := ws.copilotDirPath()
+	existingWorkspace, _ := ws.CopilotDirPath()
 	if existingWorkspace != "" {
 		return nil
 	}
 	return ws.fsUtils.Mkdir(CopilotDirName, 0755)
 }
 
-func (ws *Workspace) copilotDirPath() (string, error) {
+// CopilotDirPath returns the absolute path to the workspace's copilot dir.
+func (ws *Workspace) CopilotDirPath() (string, error) {
```

Comment:
Since this is public now we should add tests

Reply:
There are tests for it already. I've renamed it to mirror the capital letter.